### DISCUSSION TITLE
After chess960 is turned on or off, the intelligence of the ai changes

### DISCUSSION THREAD

**Discussion by jialaolian11** [fairy-stockfish/Fairy-Stockfish#676] [Q&A]
When the difficulty level -20 and chess960 are on, the ai's chess moving ideas will be in a non-intelligent phenomenon. When chess960 is off, the ai's chess moving ideas will become very intelligent. How can I modify this phenomenon

**Comment by ianfab** [comment]
If you set Skill Level to -20, then you tell the engine to play badly. If you want it to play well, then you should keep it at the maximum value of 20.

**Comment by jialaolian11** [comment]
Good morning, Sir. At present, the problem I have encountered is that after I shut down chess960, the difficulty is set to -20, according to the difficulty, its performance should be very bad, but no, its performance is very smart, may I ask where I set wrong. Here is my                 uci command setoption name Skill Level value -20     setoption name UCI_Chess960 value false    position fen ****     go depth 1
